README.md

```diff
@@ -11,6 +11,25 @@ The primary objective of this project is to enhance the scalability of ENS while
 * **Gateway**: An API capable of handling reading data from an external source following the flow specified by the [EIP-5559](https://eips.ethereum.org/EIPS/eip-5559).
 * **Client**: A module that is able to read ENS domain properties even when stored in external data sources.
 
+## Usage
+
+To run this application locally, follow these steps:
+
+1. Clone this repository to your local machine.
+2. Install dependencies by running:
+
+```bash
+yarn install
+```
+
+3. Start the development server:
+
+```bash
+yarn start
```

Comment:
Hey Lucas, i was testing the branch and i was not able to run this command (yarn start) that easy. I figure it out that it is necessary to add a .env file in the contracts folder. Maybe, it'll be good to explain this process here in the readme. It will make thinks more clear to someone that is trying to run our project.